packages/lit-element/package.json

```diff
@@ -35,7 +35,7 @@
     }
   },
   "scripts": {
-    "build": "npm run clean && tsc && rollup -c",
+    "build": "npm run clean && tsc && rollup -c && npm run build:bundle",
```

Comment:
I'm confused why there's a separate build for the bundle... it looks like you already added ability to add a list of `bundled` entrypoints in `litProdConfig` (`platform-support` uses this); is there a reason you didn't just add the build for `lit.min` there? Speed of `build:watch`?

Reply:
Actually, it looks like this may be the cause of the platform-support sauce test failures also. The standalone use of `litMonoBundleConfig` does not get the `nameCache` treatment, so `_$renderOptions` isn't being renamed to the stable property name, I believe.

Reply:
Fixed.